contracts/src/token/erc721/mod.rs

```diff
@@ -160,25 +160,29 @@ impl MethodError for Error {
     }
 }
 
-sol_interface! {
-    /// [`Erc721`] token receiver interface.
-    ///
-    /// Interface for any contract that wants to support `safe_transfers`
-    /// from [`Erc721`] asset contracts.
-    interface IERC721Receiver {
-        /// Whenever an [`Erc721`] `token_id` token is transferred
-        /// to this contract via [`Erc721::safe_transfer_from`].
+pub use receiver::*;
```

Comment:
I think we can change it back. But this hack eliminates few last warnings of rust compiler. Wdt?

Reply:
I am okay with this.
Nit: As you are introducing a new mod I would move it to another (new) file.

Reply:
Nit: Can we have just `pub use receiver::IERC721Receiver`?